src/components/VLicenseElements.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <ul>
+    <li
+      v-for="element in elements"
+      :key="element"
+      class="flex items-center gap-3 mt-2"
+    >
+      <VIcon
+        view-box="0 0 30 30"
+        :size="isSmall ? 5 : 6"
+        :icon-path="icons[element]"
+      />
+      <span class="sr-only">{{ element.toUpperCase() }}</span>
```

Comment:
For `pdm` and `cc0` or any other "single element licenses/marks" this can be left out as it's redundant with the heading of the popover.